### PR TITLE
Added slide out panel for learning resource details [#96967708]

### PR DESCRIPTION
**Pull request by carsongee**
Dependent on #235 (mostly just for the indentation).
https://www.youtube.com/watch?v=29h7WsyrwKc&feature=youtu.be

### CODE REVIEW

**Comment by pdpinch** [comment]
This is part of #163 right?

**Comment by carsongee** [comment]
Yeah

**Comment by noisecapella** [comment]
The copy to clipboard button doesn't work, in the javascript it only selects the text

**Comment by noisecapella** [comment]
Also, is there any error reporting that should happen if the client can't connect to the server? Is error handling in the client handled in a future issue?

**Comment by carsongee** [comment]
The copy to clipboard can't work with javascript, and I didn't want to include Flash to do it.  I thought it was intuitive enough to have it select, but I can change the button text.  How about "Select XML" and then maybe a balloon or something to say "Hit Cmd+C or Ctrl+C to Copy"?  Or do we want to add Flash? @Ferdi @pdpinch ?

**Comment by Ferdi** [comment]
no flash, but I think I have seen a js library that does that

**Comment by carsongee** [comment]
I added error reporting on failure

**Comment by pdpinch** [comment]
I'm fine with it as is for now. Please open an issue to deal with it later. 

---

I am mobile.

**Comment by Ferdi** [comment]
I agree with @pdpinch and maybe I was wrong about js library. It could have been flash.

File: ui/static/ui/js/main.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-define(['jquery', 'bootstrap', 'icheck'], function(jQuery) {
+define(['jquery', 'bootstrap', 'icheck'], function(rjQuery) {
```
Comment (noisecapella) [inline ui/static/ui/js/main.js]: Is rjQuery a typo?

Comment (carsongee) [inline ui/static/ui/js/main.js]: Good catch, thanks

**Comment by carsongee** [comment]
I snuck one last thing in here (can pull out if it is in the way):
image

to show descriptions if they exist, and default text if they don't on the listing page

**Comment by noisecapella** [comment]
Looks good, will merge after tests pass

**Pull request merged**